AlembicImporter/Assets/Tests/Scripts/unity.formats.alembic.tests.asmdef

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Added an assembly since tests need to befriend the alembic assemblies in order to use the internal API